frontend/pages/workspaces/[workspaceId].tsx

```
@@ -54,31 +32,36 @@ const H2 = styled.h2`
   `}
 `;
 
-interface Props {
-  workspace: {
-    title: string;
-    id: string;
-  };
-  folders: Array<Pick<Folder, "title" | "id">>;
-}
-
 const ContentWrapper = styled.div`
   display: flex;
 `;
-const WorkspaceHomepage = ({ workspace, folders }: Props) => (
-  <>
-    <Head title={workspace.title} />
-    <PageLayout>
-      <Header />
-      <ContentWrapper>
-        <Navigation workspace={workspace} folders={folders} />
-        <PageContent>
-          <MainHeading>{workspace.title}</MainHeading>
-          <H2>Most recent items</H2>
-        </PageContent>
-      </ContentWrapper>
-    </PageLayout>
-  </>
-);
 
-export default WorkspaceHomepage;
+const WorkspaceHomepage: NextPage = () => {
+  const router = useRouter();
+  const { workspaceId } = router.query;
+  const id = (workspaceId || "unknown").toString();
```

I think we should probably throw an error if there is no workspace id in the query. Ideally that would return a `404`. Not sure if that's possible.